modules/nw-configure-sysctl-interface-sriov-network-bonded.adoc

```diff
@@ -0,0 +1,227 @@
+<8> Add additional capabilities to the device. In this use case set the `type` field to `tuning`. Specify the interface-level network sysctl you want to set in the sysctl field. This examples sets all interface-level network sysctls that can be set. 
+
+. Create the bond network attachment resource:
++
+[source,terminal]
+----
+$ oc create -f sriov-bond-network-interface.yaml
+----
+
+.Verifying that the `NetworkAttachmentDefinition` CR is successfully created
+
+* Confirm that the SR-IOV Network Operator created the `NetworkAttachmentDefinition` CR by running the following command.
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions -n <namespace> <1>
+----
+<1> Replace `<namespace>` with the networkNamespace that you specified when configuring the network attachment, for example, `sysctl-tuning-test`.
++
+.Example output
+[source,terminal]
+----
+NAME                AGE 
+bond-net1           22m 
+allvalidflags       47m
+----
++
+[NOTE]
+====
+There might be a delay before the SR-IOV Network Operator creates the CR.
+====
+
+.Verifying that the additional SR-IOV network resource is successful
+
+To verify that the tuning CNI is correctly configured and the additional SR-IOV network attachment is attached, do the following:
+
+. Create a pod such as `examplepod.yaml` similar to the following:.
++
+[source,terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: tunepod
+  namespace: sysctl-tuning-test
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        {"name": "allvalidflags"}, <1>
+        {"name": "allvalidflags"},
+        {
+          "name": "bond-net1",
+          "interface": "bond0",
+          "mac": "0a:56:0a:83:04:0c", <2>
+          "ips": ["10.100.100.200/24"] <3>
+       }
+      ]
+spec:
+  containers:
+  - name: podexample
+    image: centos
+    command: ["/bin/bash", "-c", "sleep INF"]
+    securityContext:
+      runAsUser: 2000
+      runAsGroup: 3000
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: ["ALL"]
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+----
+<1> The name of the SR-IOV network attachment definition CR.
+<2> Optional: The MAC address for the SR-IOV device that is allocated from the resource type defined in the SR-IOV network attachment definition CR. To use this feature, you also must specify `{ "mac": true }` in the SriovNetwork object.
+<3> Optional: IP addresses for the SR-IOV device that is allocated from the resource type defined in the SR-IOV network attachment definition CR. Both IPv4 and IPv6 addresses are supported. To use this feature, you also must specify `{ "ips": true }` in the SriovNetwork object.
+
+. Apply the YAML: 
++
+[source,terminal]
+----
+$ oc apply -f examplepod.yaml
+----
+
+. Verify that the pod is created by running the following command:
++
+[source,terminal]
+----
+$ oc get pod
```

Comment:
I think we need to specify namespace here like with other resources 
oc get pods -n <namespace>

Reply:
fixed